internal/generate/testdata/go/deploy/olm-catalog/memcached-operator/0.0.3/memcached-operator.v0.0.3.clusterserviceversion.yaml

```diff
@@ -197,7 +197,7 @@ spec:
   - admissionReviewVersions: null
     deploymentName: memcached-operator-webhook
     failurePolicy: Fail
-    name: mmemcached.kb.io
+    generateName: mmemcached.kb.io
     rules:
     - apiGroups:
       - cache.example.com
```

Comment:
Shows that it is missing update all CSV files in : operator-sdk/internal/scorecard/alpha/testdata/